packages/pulpcore/python-dotenv/python-dotenv.spec

```diff
@@ -30,11 +34,6 @@ Summary:        %{summary}
 # Remove bundled egg-info
 rm -rf %{pypi_name}.egg-info
 
-# setup.py loads the contents of README.md and CHANGELOG.md into long_description
```

Comment:
Is this fixed?

Reply:
my local scratch builds worked, so I'd say yes.